bindata/network/ovn-kubernetes/alert-rules.yaml

```diff
@@ -15,9 +15,10 @@ spec:
     - alert: NodeWithoutOVNKubeNodePodRunning
       annotations:
         message: |
-          All nodes should be running an ovnkube-node pod, {{"{{"}} $labels.node {{"}}"}} is not.
+          All nodes except Windows nodes, should be running an ovnkube-node pod, {{"{{"}} $labels.node {{"}}"}} is not.
       expr: |
-        (kube_node_info unless on(node) kube_pod_info{namespace="openshift-ovn-kubernetes",  pod=~"ovnkube-node.*"}) > 0
+        (kube_node_info unless on(node) (kube_pod_info{namespace="openshift-ovn-kubernetes",pod=~"ovnkube-node.*"}
+        or kube_node_labels{label_node_openshift_io_os_id="Windows"})) > 0
```

Comment:
To be clear, we're not running `Windows Node exporter`  on the Windows yet. So, when we run it, should we add `label_node_openshift_io_os_id="Windows"` is that what you're implying here?

Reply:
Can you also post the output of the `kube_node_info` for the Windows and Linux nodes to make the comparison easier?

Reply:
you could use the label `kubernetes.io/os:` if I recall correctly that labels nodes by there os on startup and is not reliant on anything but kube to set it

Reply:
I used the `node.openshift.io/os_id=Windows` label because we are adding it while bootstrapping windows node with [WMCO](https://github.com/openshift/windows-machine-config-operator)

Reply:
> Can you also post the output of the `kube_node_info` for the Windows and Linux nodes to make the comparison easier?

linux nodes-
`kube_node_info{container_runtime_version="cri-o://1.19.0-31.rhaos4.6.git0a84af5.el8-dev",endpoint="https-main",instance="10.131.0.11:8443",job="kube-state-metrics",kernel_version="4.18.0-211.el8.x86_64",kubelet_version="v1.18.3+a34fde4",kubeproxy_version="v1.18.3+a34fde4",namespace="openshift-monitoring",node="ip-10-0-171-167.us-east-2.compute.internal",os_image="Red Hat Enterprise Linux CoreOS 46.82.202007161640-0 (Ootpa)",pod="kube-state-metrics-64bd56466f-vnhhq",pod_cidr="10.128.5.0/24",provider_id="aws:///us-east-2b/i-0eef758e493ff7fbb",service="kube-state-metrics"}`

windows nodes-
`kube_node_info{container_runtime_version="docker://19.3.5",endpoint="https-main",instance="10.131.0.11:8443",job="kube-state-metrics",kernel_version="10.0.17763.1282",kubelet_version="v1.18.3",kubeproxy_version="v1.18.3",namespace="openshift-monitoring",node="ip-10-0-137-99.us-east-2.compute.internal",os_image="Windows Server 2019 Datacenter",pod="kube-state-metrics-64bd56466f-vnhhq",pod_cidr="10.128.6.0/24",provider_id="aws:///us-east-2a/i-0a9a8c7124acb6e4a",service="kube-state-metrics"}`